CBPinEntryView/Classes/CBPinEntryViewDefaults.swift

```diff
@@ -12,6 +12,9 @@ struct CBPinEntryViewDefaults {
 
     // Default number of fields
     static let length: Int = 4
+    
+    //Default spacing between fileds
```

Comment:
Can you change this comment to `// Default spacing between fields`

Reply:
Yes, of course. I am sorry

Reply:
Also the spelling of fields :)

Reply:
@Fawxy I fixed it